spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -102,6 +103,12 @@
 	 */
 	private Duration connectionTimeout;
 
+	/**
+	 * Requested Channel Max. Set it to zero for unlimited. Default is 2047 from RabbitMQ
+	 * java client versions 4.7.0 and 5.3.0.
+	 */
+	private int requestedChannelMax = 2047;
```

Comment:
The default value is already documented so it shouldn't be documented again. Can you please rephrase that to rather explain in one sentence what requestedChannelMax actually is?

Reply:
More detailed description is now in updated PR, thanks for suggestion.

Reply:
It would be better to change this to `Integer` (default `null`) and add `.whenNonNull()` to the mapping; that way, if they decide to change the default in the client library, we won't need a corresponding change here.

Reply:
Thanks for suggestion, when it is set to null in RabbitProperties object it does not impact actual default in com.rabbitmq.client.ConnectionFactory, added test to verify that and updated PR.